src/constants/strings.ts

```diff
@@ -41,5 +41,6 @@ export enum Slugs {
 export const SHARING_LOCATION = '/preview/'
 
 export enum DocumentTypes {
```

Comment:
I guess this should be exported from cozy-client instead, what do you think @Crash-- ?